src/scripts/modules/media/media.coffee

```diff
@@ -32,6 +33,11 @@ define (require) ->
     summary:() -> @updateSummary()
     description: () -> @updateSummary()
 
+    events: {
+      'keydown .media-title > .title input': 'checkKeySequence'
+      'keyup .media-title > .title input': 'resetKeySequence'
+    }
```

Comment:
Just a style comment: I believe most of the CoffeeScript code has been leaving off brackets for objects when it would be unambiguous to do so.